storage/src/rocksdb_client.cpp

```diff
@@ -592,7 +596,7 @@ Status Client::createCheckpoint(const uint64_t &checkPointId) {
     if (!fs::exists(path)) {
       fs::create_directory(path);
     }
-    fs::path chkptDirPath = path / std::to_string(checkPointId);
+    fs::path chkptDirPath = getPathForCheckpoint(checkPointId);
```

Comment:
chkpt_dir_path

Reply:
Same as above - will keep for consistency in that file for now.